scipy/stats/tests/test_stats.py

```diff
@@ -3319,6 +3329,17 @@ def test_ttest_rel():
     assert_array_almost_equal(np.abs(p), pr)
     assert_equal(t.shape, (3, 2))
 
+    # test alternative parameter
+    assert_raises(ValueError, stats.ttest_rel, rvs1, rvs2, alternative="error")
+
+    t,p = stats.ttest_rel(rvs1, rvs2, axis=0, alternative="less")
+    assert_array_almost_equal(np.abs(p), 1 - pr/2)
```

Comment:
For these new tests, please get rid of all the absolute value signs. Predict what the sign should be, and include it in the test. We want the test to fail if the sign is incorrect.